src/components/Select.vue

```diff
@@ -967,7 +978,7 @@
           searchable: this.searchable,
           unsearchable: !this.searchable,
           loading: this.mutableLoading,
-          rtl: this.dir === 'rtl',
+          rtl: this.dir === 'rtl', // This can be removed - styling is handled by `dir="rtl"` attribute
```

Comment:
I'm OK to remove this now, since we no longer depend on it internally. But that could be a breaking change if someone did depend on it. Thoughts?

Reply:
@sagalbot Should we keep this class around for backward compat, or kill it?